webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/operate/template/BatchOperationTemplate.java

```diff
@@ -32,21 +26,8 @@ public class BatchOperationTemplate extends OperateTemplateDescriptor implements
   public static final String FAILED_OPERATIONS_COUNT = "failedOperationsCount";
   public static final String COMPLETED_OPERATIONS_COUNT = "completedOperationsCount";
 
-  @Autowired private OperateProperties properties;
-
-  public BatchOperationTemplate() {
-    super(null, false);
-  }
-
-  @PostConstruct
-  public void init() {
-    indexPrefix = properties.getIndexPrefix(DatabaseInfo.getCurrent());
-    isElasticsearch = DatabaseInfo.isElasticsearch();
-  }
-
-  @Override
-  public String getIndexPrefix() {
-    return properties.getIndexPrefix();
+  public BatchOperationTemplate(final String indexPrefix, final boolean isElasticsearch) {
+    super(indexPrefix, isElasticsearch);
   }
 
```

Comment:
🔧 I think here we also need to add the getVersion() correct? here will be "8.1.0" since the original was an extension of [AbstractIndexDescriptor](https://github.com/camunda/camunda/blob/b88820c5cec26171efc9a93e761705c15dab1dec/webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/AbstractIndexDescriptor.java).

Reply:
I'm happy to add it, but could you explain to me where did the `8.1.0` value come from?

Reply:
The original `BatchOperationTemplate` extents several classes until we have the `AbstractIndexDescriptor` which implements the getVersion( ) that returns "8.1.0".

Reply:
Just so that this template can return the same version as the one in operate.

Reply:
This does extend the same classes, no?
And I see here that it has `1.0.0` not `8.1.0`, that's why I'm confused

https://github.com/camunda/camunda/blob/b88820c5cec26171efc9a93e761705c15dab1dec/webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/AbstractIndexDescriptor.java#L49-L52

Reply:
![2024-10-31_14-06](https://github.com/user-attachments/assets/1d132097-f6dc-4b14-a40a-7d068d2d67e1)

That is currently running in our benchmarks. Just do bring in some data maybe this helps in the discussion